provider/provider.go

```diff
@@ -44,6 +44,9 @@ type Options struct {
 	SkipArchived bool
 	// Repository private status to display in the output if the provider is 'repository'
 	RespositoryIsPrivate bool
+	// AllRepositories collect hashes from every accessible repository,
```

Comment:
Same as above. Something like : "AllRepositories collects fileshas from every accessible repository including public ones. This is only used by "github" provider". WDYT ?